newspack-theme/archive-newspack_correction.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * The template for displaying archive pages for the newspack_correction post type
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package Newspack
+ */
+get_header();
+
+?>
+
+	<section id="primary" class="content-area">
+
+		<header class="page-header">
+
+			<span>
+
+				<h1 class="page-title"><?php esc_html_e( 'Corrections and clarifications', 'newspack-theme' ); ?></h1>
```

Comment:
```suggestion
				<h1 class="page-title"><?php esc_html_e( 'Corrections and clarifications', 'newspack' ); ?></h1>
```

This is confusing because it's different than plugins, but the Newspack Theme's textdomain doesn't need to match the theme's slug, it's just `newspack`.